getContentfulEnvironment.js

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable no-undef */
+/* eslint-disable @typescript-eslint/no-var-requires */
+const contentfulManagement = require('contentful-management');
+
+module.exports = function () {
+  const contentfulClient = contentfulManagement.createClient({
+    accessToken: 'CFPAT-9dKWzcOTDk_4RaQS4e0HslYq73KTdTSOKrb7wwADtJ8'
```

Comment:
You just created security vulnerability here. You should not put any tokens on github, learn about .env, I'm not going to steal your account but someone else would do that, especially when it comes to real project